src/service/service.go

```diff
@@ -27,9 +29,17 @@ func NewService(bindAddress string, node *node.Node, logger *logrus.Logger) *Ser
 
 func (s *Service) Serve() {
 	s.logger.WithField("bind_address", s.bindAddress).Debug("Service serving")
+
 	http.HandleFunc("/stats", s.GetStats)
+
 	http.HandleFunc("/block/", s.GetBlock)
+
+	http.HandleFunc("/graph", s.GetGraph)
+
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("src/service/static/"))))
```

Comment:
Should we embed the static files with the binaries?
Otherwise we have to worry about copying the files to the adequate location (~/src/service/static by default).
Even if we make this path configurable, it is still overhead.

Reply:
There is a bunch of tools available that are used to bundle such static assets into the final binary, i'll have a look at them and make you a proposition !